lib/Exporter.js

```diff
@@ -339,50 +337,47 @@ function callOgr (params, geojson, options, callback) {
   var outFile = params.outFile
   var metadata = options.metadata
 
-  if (fs.existsSync(outFile)) {
-    // we already have the file, just return it
-    callback(null, outFile)
+  // we already have the file, just return it
+  if (fs.existsSync(outFile)) return callback(null, outFile)
```

Comment:
Also simplified a function in `lib/Exporter` to return earlier and remove an extra layer of indentation.